src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs

```diff
@@ -8,36 +8,50 @@
 
 namespace Microsoft.VisualStudio.TestPlatform.Utilities;
 
+/// <summary>
+/// !!!NEVER USE A FLAG TO ENABLE FUNCTIONALITY!!!
+/// 
+/// The reasoning is:
+/// 
+/// * New version will automatically ship with the feature enabled. There is no action needed to be done just before release.
+/// * Anyone interested in the new feature will get it automatically by grabbing our preview.
+/// * Anyone who needs more time before using the new feature can disable it in the released package.
+/// * If we remove the flag from our code, we will do it after the feature is the new default, or after the functionality was completely removed from our codebase.
+/// * If there is a very outdated version of an assembly that for some reason loaded with the newest version of TP and it cannot find a feature flag, because we removed that feature flag in the meantime, it will just run with all it's newest features enabled.
+///
+/// Use constants so the feature name will be compiled directly into the assembly that references this, to avoid backwards compatibility issues, when the flag is removed in newer version.
+/// </summary>
+
 // !!! FEATURES MUST BE KEPT IN SYNC WITH https://github.com/dotnet/sdk/blob/main/src/Cli/dotnet/commands/dotnet-test/VSTestFeatureFlag.cs !!!
 internal partial class FeatureFlag : IFeatureFlag
 {
     private static readonly Dictionary<string, bool> FeatureFlags = new();
 
-    private const string VSTEST_FEATURE = nameof(VSTEST_FEATURE);
+    private const string VSTEST_ = nameof(VSTEST_);
 
     public static IFeatureFlag Instance { get; } = new FeatureFlag();
 
     static FeatureFlag()
     {
-        FeatureFlags.Add(ARTIFACTS_POSTPROCESSING, true);
-        FeatureFlags.Add(ARTIFACTS_POSTPROCESSING_SDK_KEEP_OLD_UX, false);
+        FeatureFlags.Add(DISABLE_ARTIFACTS_POSTPROCESSING, false);
+        FeatureFlags.Add(DISABLE_ARTIFACTS_POSTPROCESSING_NEWSDKUX, false);
     }
 
-    // Added for artifact porst-processing, it enable/disable the post processing.
+    // Added for artifact post-processing, it enable/disable the post processing.
     // Added in 17.2-preview 7.0-preview
-    public static string ARTIFACTS_POSTPROCESSING = VSTEST_FEATURE + "_" + "ARTIFACTS_POSTPROCESSING";
+    public const string DISABLE_ARTIFACTS_POSTPROCESSING = VSTEST_ + "_" + nameof(DISABLE_ARTIFACTS_POSTPROCESSING);
 
-    // Added for artifact porst-processing, it will show old output for dotnet sdk scenario.
+    // Added for artifact post-processing, it will show old output for dotnet sdk scenario.
     // It can be useful if we need to restore old UX in case users are parsing the console output.
     // Added in 17.2-preview 7.0-preview
-    public static string ARTIFACTS_POSTPROCESSING_SDK_KEEP_OLD_UX = VSTEST_FEATURE + "_" + "ARTIFACTS_POSTPROCESSING_SDK_KEEP_OLD_UX";
+    public const string DISABLE_ARTIFACTS_POSTPROCESSING_NEWSDKUX = VSTEST_ + "_" + nameof(DISABLE_ARTIFACTS_POSTPROCESSING_NEWSDKUX);
```

Comment:
I don't like `NEWSDKUX` because why do we split with underscore the other parts and not this one?

Reply:
It's longer but yeah. I cannot suggest another name because I don't understand what the feature does.

Reply:
The new console UX remove the output on console "per project" for the attachments in favor of only the final one at the end. 

This flag restore the "per project" attachment output if for some reason users parsed the output in past.